File: JurneyTag/Core/Models/City.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace JurneyTag.Models
{
    public class City
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public double MapPositionLatitude { get; set; }
        public double MapPositionLongitude { get; set; }
        public double MetersAboveSeaLevel { get; set; }
        public int Population { get; set; }
        public double Area { get; set; }
        public double PopulationDensity { get; set; }
    }
}

File: JurneyTag/ClientApp/src/app/Models/City.ts
import { MapLocation } from "./Location";

export interface City {
    id : number;
    name : string;
    description : string;
    metersAboveSeaLevel : number;
    population : number;
    area : number;
    populationDensity: number;
    location : MapLocation;
    mainImage : any;
}